$OpenBSD: patch-tests_CMakeLists_txt,v 1.1 2015/01/05 22:30:54 dcoppa Exp $

Fix for the following error:

<command-line>:0:13: warning: missing terminating " character [enabled by default]
/usr/ports/pobj/marble-4.14.2/marble-4.14.2/tests/TestGeoDataWriter.cpp:52:5: error: stray '\' in program
     dataDir = QDir( TESTSRCDIR );
     ^
/usr/ports/pobj/marble-4.14.2/marble-4.14.2/tests/TestGeoDataWriter.cpp:52:5: error: missing terminating " character


--- tests/CMakeLists.txt.ports.orig	Mon Oct  6 00:35:01 2014
+++ tests/CMakeLists.txt	Mon Jan  5 12:07:23 2015
@@ -35,7 +35,7 @@ endif ()
 # use this for accessing TESTSRCDIR/data
 #############################################################
 
-add_definitions( -DTESTSRCDIR=\\"${CMAKE_CURRENT_SOURCE_DIR}\\" )
+add_definitions( -DTESTSRCDIR="\\"${CMAKE_CURRENT_SOURCE_DIR}\\"" )
 if (NOT QTONLY)
   add_definitions( -DQT_GUI_LIB )
 endif ()
